boards/arm/v2m_musca_b1/doc/index.rst

```diff
@@ -0,0 +1,421 @@
+register. The purpose of this register is to allow individual control of
+clocks, reset-signals and interrupts to peripherals, and pin-muxing.
+
+QSPI boot memory
+================
+Normal Musca-B1 test chip boot operation is from 4MB eFlash by default, and
+it offers the fastest boot method.
+Musca-B1 test chip also support to boot from 8MB QSPI. You can update the
+DAPLink firmware for either QSPI or eFlash for booting.
+
+Programming and Debugging
+*************************
+
+Musca B1 supports the v8m security extension, and by default boots to the
+secure state.
+
+When building a secure/non-secure application, the secure application will
+have to set the idau/sau and mpc configuration to permit access from the
+non-secure application before jumping.
+
+The following system components are required to be properly configured during the
+secure firmware:
+
+- AHB5 TrustZone Memory Protection Controller (MPC).
+- AHB5 TrustZone Peripheral Protection Controller (PPC).
+- Implementation-Defined Attribution Unit (IDAU).
+
+For more details please refer to `Corelink SSE-200 Subsystem`_.
+
+Flashing
+========
+
+DAPLink
+---------
+
+V2M Musca B1 provides:
+
+- A USB connection to the host computer, which exposes a Mass Storage and an
+  USB Serial Port.
+- A Serial Flash device, which implements the USB flash disk file storage.
+- A physical UART connection which is relayed over interface USB Serial port.
+
+This interfaces are exposed via DAPLink which provides:
+
+- Serial Wire Debug (SWD).
+- USB Mass Storage Device (USBMSD).
+- UART.
+- Remote reset.
+
+For more details please refer
+to the `DAPLink Website`_.
+
+
+Building a secure only application
+----------------------------------
+
+
+You can build applications in the usual way. Here is an example for
+the :ref:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: v2m_musca_b1
+   :goals: build
+
+Open a serial terminal (minicom, putty, etc.) with the following settings:
+
+- Speed: 115200
+- Data: 8 bits
+- Parity: None
+- Stop bits: 1
+
+Reset the board, and you should see the following message on the corresponding
+serial port:
+
+.. code-block:: console
+
+   Hello World! musca_b1
+
+Building a secure/non-secure with Trusted Firmware
+--------------------------------------------------
+
+The process requires five steps:
+
+1. Build Trusted Firmware (tfm).
+2. Import it as a library to the Zephyr source folder.
+3. Build Zephyr with a non-secure configuration.
+4. Merge the two binaries together and sign them.
+5. Concatenate the bootloader with the signed image blob.
+
+In order to build tfm please refer to `Trusted Firmware M Guide`_.
+Follow the build steps for AN521 target while replacing the platform with
+``-DTARGET_PLATFORM=MUSCA_B1`` and compiler (if required) with ``-DCOMPILER=GNUARM``
+
+Copy over tf-m as a library to the zephyr project source and create a shortcut
```

Comment:
Should this be ``tfm`` instead of ``tf-m``?
```suggestion
Copy over tf-m as a library to the Zephyr project source and create a shortcut
```

Reply:
Updated, Thanks